client/cody/src/services/LocalAppDetector.ts

```diff
@@ -0,0 +1,113 @@
+import * as vscode from 'vscode'
+
+const LOCAL_APP_LOCATIONS: { [key: string]: string[] } = {
+    darwin: [
+        '~/Library/Application Support/com.sourcegraph.cody',
+        '/Applications/Sourcegraph App.app/Contents/MacOS/Sourcegraph App',
```

Comment:
```suggestion
        '/Applications/Sourcegraph.app',
```
I think we can shorten the path to just the `.app` part? Aside from that we'll need to come back to this when we [rename the app](https://github.com/sourcegraph/sourcegraph/issues/53120) to `Cody.app` in upcoming releases.

Reply:
That's a good point! I will also add a TODO on this. Or should we check both and if they have either one we they can pass?

Reply:
Yeah we could add both right now. The intention of the paths list was to have locations that we scan until we find one, so this would makes sense.
```suggestion
        '/Applications/Sourcegraph.app',
        '/Applications/Cody.app',
```